src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,37 +1,36 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
 import ColorPickerPalette from './ColorPickerPalette';
-import { AnnotationMode } from '../annotations/types';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerControl.scss';
 
+const colors = Object.values(AnnotationColor);
+
 export type Props = {
-    annotationMode?: AnnotationMode;
-    onAnnotationColorClick: (color: string) => void;
+    annotationColor?: string;
     isActive?: boolean;
+    onColorSelect: (color: string) => void;
 };
 
 export default function ColorPickerControl({
-    annotationMode,
+    annotationColor = bdlBoxBlue,
     isActive = false,
-    onAnnotationColorClick,
+    onColorSelect,
     ...rest
 }: Props): JSX.Element | null {
     const [isColorPickerToggled, setIsColorPickerToggled] = useState(false);
 
-    if (annotationMode !== AnnotationMode.DRAWING) {
-        return null;
-    }
+    const handleSelect = (color: string): void => {
+        setIsColorPickerToggled(false);
+        onColorSelect(color);
+    };
 
     return (
         <div className="bp-ColorPickerControl">
             {isColorPickerToggled && (
                 <div className="bp-ColorPickerControl-palette">
```

Comment:
Not related to this PR, but we'll need to address the accessibility portion of this control prior to release. There's no association between the toggle button and the palette right now, nor is there any keyboard handling for the palette buttons other than tab/shift+tab.

Reply:
Thanks for the callout, I've added a ticket to track the progress for that.